pyrefinebio/high_level_functions.py

```diff
@@ -229,3 +228,38 @@ def download_quandfile_compendium(
         extract=extract,
         prompt=prompt
     )
+
+
+def create_token(silent=False):
```

Comment:
```suggestion
def create_token(agree_to_terms=None, save_token=None):
```

I think if someone passes in `None` we should prompt, but otherwise just use `True` or `False`.

I think that we should do this like this so that someone can use this function to create and activate a token without saving it nor having it interrupt their script: `create_token(True, False)`

(If I'm messing around with a new tool to see if I like it and/or it does what I want, I don't necessarily want to have it leave files in my home directory.)

Reply:
for the CLI should I just leave the one silent flag?
If they don't save with the cli the token will just disappear forever.